composer.json

```diff
@@ -9,6 +9,7 @@
         "drupal/diff": "^1.0",
         "drupal/responsive_preview": "^2.0",
         "drupal/scheduled_transitions": "^2.1",
+        "drupal/symfony_mailer": "^1.4",
```

Comment:
This might be minor as we have symfony_mailer installed, but are we sure we want to include the dependency here?

I see some new config for lando and mailhog which feels like it needs to be moved into profile or own module maybe as email config might be used in lots of places. 

It would be good to get a sense of others website email setup and if they could be supported (some might be using Gov Notifiy for their email).

I see there is a plugin for Email notifications that uses the new email system that Symfony mailer provides. Do we need to include the dependency for the plugin to work, or could it be a suggestion and dev dependency for the tests?

Reply:
Although...
```
$ composer why symfony/mailer
drupal/core             10.2.2 requires  symfony/mailer (^6.4)    
drupal/core-recommended 10.2.2 requires  symfony/mailer (~v6.4.0) 
symfony/http-kernel     v6.4.3 conflicts symfony/mailer (<5.4)    
symfony/mime            v6.4.3 conflicts symfony/mailer (<5.4)  
```

Reply:
The Symfony Mailer module provides a nice way to edit email template files. It's also easy to add new ones should we wish to add notifications on other events.

I can use the symfony/mailer component included with Drupal, but we would either need to hard code the email templates or implement our own templating system.

As it stands this work assumes the Drupal Symfony module is installed and provides some config for it. It should be possible to work with either, but I would need to investigate.

Do others have opinions on whether we should require this module?

Reply:
> This is the Drupal Symfony Mailer module (https://www.drupal.org/project/symfony_mailer) rather than the Symfony library that is now included with Drupal core.

Got it now.  When I was testing this functionality today, apart from review reminder emails, I also received a system update reminder from the core "update" module.  That's when I realized (something Andy explains above already) the Symfony mailer module has become the site-wide mail processor.  I am trying to understand if this maybe lead to conflict if a site is already using other mail related modules such as the smtp module.

I agree with Andy when he says above "...feels like it needs to be moved into profile or own module maybe as email config might be used in lots of places."  For now, keeping Symfony mailer as a "dev" dependency of this module as per Andy's suggestion may not be a bad idea.